### PR TITLE
removed $ from start of commands

### PR DESCRIPTION
**Pull request by agduncan94**
https://github.com/ga4gh/dockstore/issues/1838

### CODE REVIEW

File: src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts [hunk 1]
```diff
@@ -118,15 +118,15 @@ exec newgrp docker
 #### Part 4 - Confirm installation
 1. Run our dependencies to verify that they have been installed properly.
 \`\`\`
-$ dockstore --version
+dockstore --version
```
Comment (denis-yuen) [inline src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts]: hmm, unclear this way which lines are commands and which lines are their output

Comment (agduncan94) [reply src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts]: Good point, do you think I should readd the $ or do something else?

Comment (denis-yuen) [reply src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts]: Not sure, may need some experimentation to see what looks best or to look at other projects with documentation online. 

Brainstorming
1. Re-add $
2. Don't use a quote block and distinguish the commands from the output in some other way (italics?)

Comment (agduncan94) [reply src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts]: I'll look at some coding tutorial sites, maybe they have some good implementations.

**Pull request merged**